### DISCUSSION TITLE
Displaying google analytics report in docusaurus website

### DISCUSSION THREAD

**Discussion by Karan-S-dev** [facebook/docusaurus#7812] [Q&A] [answered]
I have connected my site with google analytics using,

module.exports = {
  presets: [
    [
      '@docusaurus/preset-classic',
      {
        gtag: {
          trackingID: 'G-226F0LR9KE',
          anonymizeIP: true,
        },
      },
    ],
  ],
};

Now I want to display the analyics records or report in my website in a seperate page. Is there a solution for that?

Thanks in advance

**Comment by slorber** [comment, marked as answer]
No solution in Docusaurus: you should create a custom React page and implement it yourself. This is not a feature we plan to implement

**Reply by Karan-S-dev** [reply]
Okay Thanks
